__tests__/help.ts

```diff
@@ -20,3 +22,15 @@ test('npx unimported --help', (done) => {
     done();
   });
 });
+
+test('npx unimported --help', (done) => {
+  // note about `./` path: jest executes the tests from the root directory
+  // wrong file name
+  exec('node ./bin/1unimported.js --help', (error, stdout, stderr) => {
```

Comment:
If the purpose was to promisify `exec`, should the tests be converted to an `async` function and we `await` the result of it instead of using the callback?